build-system/tasks/check-links.js

```diff
@@ -134,7 +134,8 @@ function checkLinksInFile(file) {
         return;
       }
       let containsDeadLinks = false;
-      for (const {link, status, statusCode} of results) {
+      // eslint-disable-next-line prefer-const
+      for (let {link, status, statusCode} of results) {
```

Comment:
Please undo this change.

Reply:
It's a legitimate problem, thanks for bringing it to my attention.

I'm fixing it individually in #27197.